docs/csharp/roslyn-sdk/tutorials/how-to-write-csharp-analyzer-code-fix.md

```diff
@@ -13,27 +13,8 @@ In this tutorial, you'll explore the creation of an **analyzer** and an accompan
 
 ## Prerequisites
 
-> [!NOTE]
```

Comment:
I'll defer to @IEvangelist whether he wants to completely delete the note or use my suggestion in https://github.com/dotnet/docs/issues/19332#issuecomment-654344262.

I'm okay with both options as long as the article strictly mentions 16.7 as prerequisite.

Reply:
I suppose, it could still be useful to call this out. But the note is so distracting, I think I prefer the explicit prerequisite.

Reply:
@IEvangelist, I agree that it's too long and could be distracting.
The only thing left is to actually make sure this works well with 16.7 and everything is fixed. I remember seeing an issue here complaining about problems and he stated using 16.7. But I don't know if that problem is from user-side or not.